tests/TestCase/Mailer/EmailTest.php

```diff
@@ -2712,6 +2712,7 @@ public function testJsonSerialize()
         ];
         $this->assertEquals($expected, $result);
 
+        debug(unserialize(serialize($this->CakeEmail)));
         $result = json_decode(json_encode(unserialize(serialize($this->CakeEmail))), true);
```

Comment:
I don't remember why this json parity was important. Was it used to move email object across process boundaries?